### ISSUE TITLE
Subscription still owned by previous Parent Company.

### ISSUE THREAD

**Issue by btfrise** [Rise-Vision/rise-vision-apps#1524]
Company [CCAO](https://apps.risevision.com/displays/list?cid=43472e21-164c-408f-b341-4263ba78115a) requested to be removed from [Palmer Conservation Consulting](https://apps.risevision.com/billing?cid=1dd115bf-2536-4517-af0e-42f01332c19a) company structure. Subscriptions were cancelled.

After moving CCAO away from Parent Company, it retains _Subscription managed by Palmer Conservation Consulting_ and cannot create or edit subscriptions.

**Comment by tejohnso** [comment]
@btfrise Moving the company from previous parent to the Rise Vision company doesn't not change subscription ownership.  That is done through Account & Billing and as seen ![here](https://user-images.githubusercontent.com/1511535/75187964-40222580-5719-11ea-94f5-c0474be6045e.png) from the [account and billing](https://apps.risevision.com/billing?cid=1dd115bf-2536-4517-af0e-42f01332c19a) page, Palmer Conservaction Consulting still has that active subscription for CCAO as far as datastore is concerned.  However, if you click edit, or look in Chargebee, you'll see the subscription was cancelled in January.

I think what you are seeing is a case of https://github.com/Rise-Vision/store-server/issues/1114

So, to summarize:

1. The company move worked, CCAO parent is now Rise Vision Inc (f114ad26-949d-44b4-87e9-8528afc76ce4)
2. The subscription is incorrect due to an issue in store-server that left some data out of sync, which we'll be correcting

If okay with you, I'd like to close this as an instance of https://github.com/Rise-Vision/store-server/issues/1114

**Comment by tejohnso** [comment]
@btfrise as discussed, I've updated CCAO directly since they're waiting to be able to pay for their own subscription. As of now they'll no longer see the "Subscription managed by Palmer Conservation Consulting" message.  I think we can close this. Let me know if you have any questions / concerns
